orca-5/Dockerfile

```diff
@@ -47,7 +48,6 @@ nextgenmap \
 ngmaster \
 nonpareil \
 novoalign \
-ntcard \
```

Comment:
You can put this line back in. It only fails if the formula is out of date. It'll be up to date after `brew upgrade ntcard`.
```console
$ brew install hello; echo $?
Warning: hello 2.10 is already installed and up-to-date
To reinstall 2.10, run `brew reinstall hello`
0
```

Reply:
Right makes sense -- I just triggered the new build but I'll revert all the changes I made in this PR after it (hopefully) succeeds. It was just weird because of the time between successful builds.